boards/arm/stm32l562e_dk/doc/index.rst

```diff
@@ -0,0 +1,283 @@
+  - OCTOSPI memory interface
+
+- Rich analog peripherals (independent supply)
+
+  - 3x 12-bit ADC 5 MSPS, up to 16-bit with hardware oversampling, 200 uA/MSPS
+  - 2x 12-bit DAC, low-power sample and hold
+  - 2x operational amplifiers with built-in PGA
+  - 2x ultra-low-power comparators
+  - 4x digital filters for sigma delta modulator
+
+- 19x communication interfaces
+
+  - USB Type-C / USB power delivery controller
+  - 2.0 full-speed crystal less solution, LPM and BCD
+  - 2x SAIs (serial audio interface)
+  - 4x I2C FM+(1 Mbit/s), SMBus/PMBus
+  - 6x USARTs (ISO 7816, LIN, IrDA, modem)
+  - 3x SPIs (7x SPIs with USART and OCTOSPI in SPI mode)
+  - 1xFDCAN
+  - 1xSDMMC interface
+  - 2x 14 channel DMA controllers
+
+- CRC calculation unit
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell |trade|
+
+
+More information about STM32L562QE can be found here:
+
+- `STM32L562QE on www.st.com`_
+- `STM32L562 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr stm32l562e_dk board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/stm32l562e_dk/stm32l562e_dk_defconfig``
+
+
+Connections and IOs
+===================
+
+STM32L562E-DK Discovery Board has 8 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+For mode details please refer to `STM32L562E-DK Discovery board User Manual`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- USART_1 TX/RX : PA9/PA10
+- USART_2 TX/RX : PD5/PD6
+- LPUART_1 TX/RX : PB11/PB10
+- I2C_1 SCL/SDA : PB6/PB7
+- SPI_1 SCK/MISO/MOSI/NSS : PG2/PG3/PG4/PG5
```

Comment:
These don't semm to be supported on this board yet

Reply:
The documentation has been updated to remove the items that are not yet supported.